project.properties

```diff
@@ -12,4 +12,4 @@
 
 android.library=true
 # Project target.
-target=android-14
+target=android-23
```

Comment:
Not completely sure of the android etiquette here - is there a reason to keep this "up to date"? I originally thought the best approach was "keep this number as low as possible" for backwards compatibility reasons.